src/tests/multihost/ad/test_adparameters.py

```diff
@@ -756,3 +756,52 @@ def test_0018_bz1734040(multihost, adjoin, create_aduser_group):
         client.restore_sssd_conf()
 
         assert find.search(log), "Expected log record is missing."
+
+    @pytest.mark.tier1
+    def test_0019_bz2070189(self, multihost, adjoin):
+        """
+        :title: sssd error triggers backtrace :
```

Comment:
a colon `:` is not needed at the end of the test title. I dont know if it will impact the import of test case or results